### PR TITLE
AND conditional to work on Linux

### PR DESCRIPTION
**Pull request by pedrocesarti**

### CODE REVIEW

File: setup [hunk 1]
```diff
@@ -17,7 +17,7 @@ if test "$(uname)" = "Darwin"; then
     brew upgrade ansible
   fi
 elif test "$(uname)" = "Linux"; then
-  if test -f /etc/lsb-release -a ! "$(which ansible)"; then
+  if test -f /etc/lsb-release && -a ! "$(which ansible)"; then
```
Comment (caarlos0) [inline setup]: I didn't understand wha't you'r trying to do here.

The `-a` is already an `AND`, the `&&` before it will actually break the `if` clause...

Comment (pedrocesarti) [reply setup]: Actually I don't know why when I'm running on Linux the argument `-a` isn't recognize.

```shell
./setup.sh: line 20: test: argument expected
Not supported yet.

GNU bash, version 4.3.46(1)-release (x86_64-pc-linux-gnu)
```

Only runs when I add the AND.

P.S: Sorry about the PR, the flag `-a` shouldnt stay.

Comment (caarlos0) [reply setup]: hmm, some shells may indeed not recognize the `-a`, but AFAIK bash wasn't one of them.

Anyways, removing the `-a` will turn it into a valid `if` statement again, yes :)

**Comment by pedrocesarti** [comment]
Sorry, I forgot the test. Now it's working.

Thank you man!!

**Comment by caarlos0** [comment]
Np, thanks ;)_

**Pull request merged**